lib/services/migrations_test.go

```diff
@@ -240,3 +240,58 @@ func (s *MigrationsSuite) TestMigrateCertAuthorities(c *C) {
 	in.AllowedLogins = nil
 	c.Assert(out3, DeepEquals, *in)
 }
+
+func (s *MigrationsSuite) TestMigrateRoles(c *C) {
```

Comment:
I don't see migrations in the database launched when Teleport starts. Where they are?